python/tests/test_vmap.py

```diff
@@ -49,8 +49,10 @@ def test_unary(self):
             "sin",
             "sqrt",
             "square",
+            "degrees",
+            "radians",
         ]
-        ops = ["erfinv"]
+        ops += ["erfinv"]
```

Comment:
Damn thanks for catching that. You could also do it as follows though
```suggestion
                "erfinv"
        ]
```

Reply:
Apparently `"erfinv"` is already there in the previous list, so I think we can just remove it from this line.